scripts/deploy_assisted_service.sh

```diff
@@ -117,9 +117,15 @@ else
     print_log "Updating assisted_service params"
 
     if [ "${DEBUG_SERVICE}" == "true" ]; then
-        print_log " Patching assisted service image with a debuggable code "
-        (cd assisted-service/ && skipper --env-file ../skipper.env make update-local-image -e LOCAL_ASSISTED_ORG=$(minikube ip):5000 -e CONTAINER_BUILD_EXTRA_PARAMS="--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file")
+        # Forward the minikube registry addon k8s service to the host to push the debug image using localhost:5000
+        kubectl port-forward service/registry -n kube-system 5000:80 > /dev/null 2>&1 &
```

Comment:
Work with xinetd like we do for the rest of the port-forwarding with `spawn_port_forwarding_command`.

Reply:
why? It shouldn't stay open

Reply:
To be aligned with all the rest of the post-forwarding solutions.

Reply:
anyway added to the xinetd